tests/app/celery/test_tasks.py

```diff
@@ -1733,7 +1733,7 @@ def test_save_api_email_or_sms(mocker, sample_service, notification_type):
         data.update({"to": "+447700900855"})
         expected_queue = QueueNames.SEND_SMS
 
-    encoded = signing.encode(data)
+    encoded = signing.encode(data | {"template_has_unsubscribe_link": False})
```

Comment:
if we do need to keep these tests, then i think it'd be a bit more readable to just include `"template_has_unsubscribe_link": False` in the main data block 🤔

Reply:
I will remove them entirely as they will be redundant soon any way, as `save_api_email` is about to be removed.